source/localizable/object-model/classes-and-instances.md

```diff
@@ -212,10 +211,17 @@ and [`set()`][8] accessor methods:
 [8]: http://emberjs.com/api/classes/Ember.Object.html#method_set
 
 ```js
-var person = Person.create();
+Person = Ember.Object.extend({
+  name: 'Robert Jackson'
```

Comment:
@locks In every section, before each instance of `Person` is created, the class is first defined (e.g. [line 135](https://github.com/emberjs/guides/pull/1399/files#diff-ef8f883b306b4c47672331305e5d5da6R135) and [line 181](https://github.com/emberjs/guides/pull/1399/files#diff-ef8f883b306b4c47672331305e5d5da6R181)). To maintain consistency I thought it would make sense to expand this code block as well.